### DISCUSSION TITLE
Not able to execute az login

### DISCUSSION THREAD

**Discussion by sudhakatta** [learndevopsonline/learndevopsonline#257] [Q&A] [answered]
Hi @r-devops /Venkat,

I have just finished automation of individual scripts and started with the VM creation via az CLI.
I executed az login --use-device-code, It is successful on web browser as below. 
![image](https://user-images.githubusercontent.com/125816794/224559967-4efb98fc-ad48-4a94-a532-65786a05a8b9.png)
but in the terminal it shows the below error and could not move further
![image](https://user-images.githubusercontent.com/125816794/224559988-3dd2a627-0c45-4a36-8672-f2e301fe3ede.png)

Kindly advice.

**Comment by akai257** [comment]
We can debug this on session, use the inbuilt shell in Azure

**Reply by akai257** [reply]
![image](https://user-images.githubusercontent.com/23319383/224560571-8374ce8c-860f-44bc-8cd9-578da59e9462.png)

**Comment by sudhakatta** [comment, marked as answer]
@akai257 Thanks for Prompt response.
So shall I go head and create this?
![image](https://user-images.githubusercontent.com/125816794/224561118-75628ad7-4d44-44da-a173-9b35914082b9.png)
![image](https://user-images.githubusercontent.com/125816794/224561142-2f0d2773-918e-434c-b910-189b1ee147a4.png)

**Reply by akai257** [reply]
Yes
